docs/client_advanced.rst

```diff
@@ -26,27 +26,35 @@ Custom Request Headers
 If you need to add HTTP headers to a request, pass them in a
 :class:`dict` to the *headers* parameter.
 
-For example, if you want to specify the content-type for the previous
-example::
```

Comment:
I did not find `previous example`

Reply:
I think it was moved to quickstart section.